### ISSUE TITLE
Investigate entities that might have multiple CRD numbers

### ISSUE THREAD

**Issue by aepyornis** [public-accountability/littlesis-rails#956]
Right now _Fidelity_ (Unmatched Item #5279) and _Commonwealth Equity Services, LLC_ (Unmatched Item #7671) create matching errors

**Comment by aepyornis** [comment]
```
Completed 500 Internal Server Error in 14ms (ActiveRecord: 6.4ms)
ExternalDatasetService::InvalidMatchError (Entity 28419 already has a crd_number. Cannot match row1076):
app/services/external_dataset_service/iapd.rb:18:in `validate_match!'
app/services/external_dataset_service.rb:9:in `validate_match!'
app/models/external_dataset.rb:46:in `match_with'
app/controllers/external_datasets_controller.rb:33:in `match'
```

**Comment by aepyornis** [comment]
resolved by #1022

Entities may now have multiple CRD numbers.
